test/ez-content-view.html

```diff
@@ -17,20 +17,22 @@
         <test-fixture id="BasicTestFixture">
             <template>
                 <ez-content-view>
-                    <ul class="ez-tabs-list">
-                        <li class="ez-tabs-label is-tab-selected"><a href="#tab1">Tab 1</a></li>
-                        <li class="ez-tabs-label"><a href="#tab2" class="tab-link">Tab 2</a></li>
-                        <li class="ez-tabs-label"><a href="#async-tab" class="async-tab-link">Async tab</a></li>
-                    </ul>
-                    <div class="ez-tabs-panels">
-                        <div class="ez-tabs-panel is-tab-selected" id="tab1">
-                            <a href="#whatever" class="not-tabs-label">Not tabs label link</a>
-                            Some content
+                    <div class="ez-tabs">
+                        <ul class="ez-tabs-list">
+                            <li class="ez-tabs-label is-tab-selected"><a href="#tab1">Tab 1</a></li>
+                            <li class="ez-tabs-label"><a href="#tab2" class="tab-link">Tab 2</a></li>
+                            <li class="ez-tabs-label"><a href="#async-tab" class="async-tab-link">Async tab</a></li>
+                        </ul>
+                        <div class="ez-tabs-panels">
+                            <div class="ez-tabs-panel is-tab-selected" id="tab1">
+                                <a href="#whatever" class="not-tabs-label">Not tabs label link</a>
+                                Some content
+                            </div>
+                            <div class="ez-tabs-panel" id="tab2">
+                                Some other content
+                            </div>
+                            <ez-asynchronous-block id="async-tab"></ez-asynchronous-block>
                         </div>
-                        <div class="ez-tabs-panel" id="tab2">
-                            Some other content
-                        </div>
-                        <ez-asynchronous-block id="async-tab"></ez-asynchronous-block>
```

Comment:
since `mixins/js/ez-tabs.js` is also included in this file, you need to add the `SeveralTabsSeriesTestFixture` here as well (but make sure to use `ez-content-view` not `ez-test-tabs`)